biblatex-iso690.tex

```diff
@@ -471,15 +599,21 @@ \subsubsection{Edition}
 
 \subsubsection{Date}
 
+If an exact date is not known, an approximate date should be supplied in
+brackets preceded by circa string (ca.). To achieve this behaviour, specify
+the date followed by a tilde, as in the example below.
+
+\begin{ltxexample}
+  date = {1490~}         % tilde meaning circa
+\end{ltxexample}
+
 In case no date is given and also no approximation is possible, that
 should be stated. Please reflect this in the \t|bib| file by including
 the following line in the respective entry.
 
-\begin{verbatim}
-  ..
+\begin{ltxexample}
   date = {\mkbibbrackets{\bibsstring{nodate}}}
```

Comment:
Biber will choke on this or at least complain. Better to do leave nodate dates empty and test for empty dates in the style. Something like https://golatex.de/viewtopic,p,109704.html#109704 (sorry, it's in German, but the important bit is th code.)